public/script.js

```diff
@@ -1,28 +1,32 @@
-$(document).ready(function() {
-    $.ajax({
-        url: '/posts',
-        dataType: 'json',
-        success: function(data) {
+document.onreadystatechange = function () {
+  if (document.readyState === 'complete') {
+    var xhr = new XMLHttpRequest();
+    xhr.onreadystatechange = function() {
+      if(xhr.readyState === 4){
+        if (xhr.status === 200){
+          var data = JSON.parse(xhr.responseText);
+          for (var blogPost in data) {
+            var postDiv         = document.createElement('div');
+            var postText        = document.createElement('p');
+            var thumbnail       = document.createElement('img');
+            var postContainer   = document.getElementsByClassName('post-container')[0];
 
-            for (var blogPost in data) {
-                var postDiv         = document.createElement('div');
-                var postText        = document.createElement('p');
-                var thumbnail       = document.createElement('img');
-                var postContainer   = document.getElementsByClassName('post-container')[0];
+            thumbnail.src = "./img/logo2.png";
+            thumbnail.className = "thumbnail";
+            postText.innerHTML = data[blogPost];
+            postDiv.className = "post";
 
-                thumbnail.src = "./img/logo2.png";
-                thumbnail.className = "thumbnail";
-                postText.innerHTML = data[blogPost];
-                postDiv.className = "post";
-
-                postDiv.appendChild(thumbnail);
-                postDiv.appendChild(postText);
-                postContainer.appendChild(postDiv);
-                
-            }
-        },
-        error: function(error){
-            console.log(error);
+            postDiv.appendChild(thumbnail);
+            postDiv.appendChild(postText);
+            postContainer.appendChild(postDiv);
+          }
+        }
+        else {
+          console.log(error);
         }
-    });
-});
+      }
+    }
+    xhr.open('GET', '/posts', true);
```

Comment:
This is up to you, but `true` is the default value for `xhr.open` anyway, so passing it in is a bit redundant

Reply:
please review pull request #54 :smile: